.github/workflows/sync-flow-go.yml

```
@@ -0,0 +1,70 @@
+# Open a PR in flow-go when a PR in cadence is merged
+# The PR in flow-go is based on the last opened `auto-cadence-upgrade/*` if it exists, otherwise on the master branch
+
+
+# Only run on pull requests merged to master
+on:
+  pull_request:
+    branches:
+      - master
+    types: [closed]
+
+jobs:
+  sync-flow-go:
+    runs-on: ubuntu-latest
+    # the PR could have been closed otherwise. Only run if it has actually ben merged
+    if: github.event.pull_request.merged == true
```

Maybe this can be simplified to just run on each push to master?

But then I guess the events at the bottom (`github.event.pull_request`) aren't available / would need to change?


```suggestion
  push:
    branches:
      - master
      
jobs:
  sync-flow-go:
    runs-on: ubuntu-latest
```

---

The convenience of having a reference of a Cadence PR in the generated flow-go PR would be lost... Not sure that would be the best approach.

Are there pushes to master that happen without PRs?